### DISCUSSION TITLE
Reg ProductMediaCreate

### DISCUSSION THREAD

**Discussion by chikkugadu** [saleor/saleor#7831] [Q&A]
![Untitled](https://user-images.githubusercontent.com/58038474/129311878-fefd35fa-7982-47c6-bbcb-4a0beff5aefe.png)

can some help me regarding productmedia upload .

**Comment by karolkielecki** [comment]
@chikkugadu Hi, please provide more information, e.g., your Saleor version; what would you like to upload--is it a video or what? If you haven't seen our documentation regarding this mutation, here you go: https://docs.saleor.io/docs/3.0/developer/api-reference/objects/product-media-create/
